### PR TITLE
[IA-1332] Jc update build for docs

### PR DESCRIPTION
**Pull request by jdcanas**
This conflicts with my other PR, but the merge will be trivial as I included the build.sh changes from that PR in this file as well, and simply added to it.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -2,17 +2,34 @@
 # Run `gcloud auth configure-docker --quiet` before running this script
 # Example: ./build.sh terra-jupyter-base
 
+PYTHON_IMAGES=("terrra-jupyter-hail" "terra-jupyter-python" "terra-jupyter-base" "terra-jupyter-bioconductor")
```
Comment (Qi77Qi) [inline build.sh]: bioconductor has both python and R

File: build.sh [hunk 2]
```diff
@@ -23,6 +40,31 @@ docker image build ./$IMAGE_DIR --tag $REPO/$IMAGE_DIR:$TAG_NAME --tag $REPO/$IM
...
+if [ $(contains "${R_IMAGES[@]}" "$IMAGE_DIR") == "true" ]; then
+    #R is a wonderful language that requires you to execute code within a shell to find the packages
+    docker exec $IMAGE_DIR sh -c "echo 'ip <- as.data.frame(installed.packages()[,c(1,3:4)]) \n rownames(ip) <- NULL \n ip <- ip[is.na(ip\$Priority),1:2,drop=FALSE] \n print(ip, row.names=FALSE)' > temp.r; Rscript temp.r; rm temp.r" > $R_OUTPUT_FILE
+    echo "gsutil cping $R_OUTPUT_FILE to $DOCUMENTATION_BUCKET"
+    gsutil cp $R_OUTPUT_FILE $DOCUMENTATION_BUCKET
+fi
+
+docker kill $IMAGE_DIR
+docker image rm -f $REPO/$IMAGE_DIR:$VERSION
+docker image rm -f $REPO/$IMAGE_DIR:$TAG_NAME
```
Comment (Qi77Qi) [inline build.sh]: what images this remove?

File: build.sh [hunk 3]
```diff
@@ -23,6 +40,31 @@ docker image build ./$IMAGE_DIR --tag $REPO/$IMAGE_DIR:$TAG_NAME --tag $REPO/$IM
     && docker push $REPO/$IMAGE_DIR:$TAG_NAME \
     && docker push $REPO/$IMAGE_DIR:$VERSION
 
-docker rm $REPO/$IMAGE_DIR
+#update remote documentation
+#will fail if you are not gcloud authed as dspci-wb-gcr-service-account
+    #docker run --rm  -v ~/.vault-token:/root/.vault-token:ro broadinstitute/dsde-toolbox:latest vault read --format=json secret/dsde/dsp-techops/common/dspci-wb-gcr-service-account.json | jq .data > dspci-wb-gcr-service-account.json
+    #gcloud auth activate-service-account --key-file=dspci-wb-gcr-service-account.json
```
Comment (akarukappadath) [inline build.sh]: are these meant to stay commented out?

Comment (jdcanas) [reply build.sh]: I updated this in another PR but I think this got applied as a change over the rebase. Thanks

File: build.sh (hunk 2 above)
Comment (jdcanas) [reply build.sh]: it deletes the local images so they don't take up disk space on the jenkins node

**Pull request merged**